substrate/primitives/staking/src/lib.rs

```diff
@@ -304,6 +310,27 @@ pub trait StakingInterface {
 	fn set_current_era(era: EraIndex);
 }
 
+/// Set of low level apis to manipulate staking ledger.
+///
+/// These apis bypass some or all safety checks and should only be used if you know what you are
+/// doing.
+pub trait StakingUnsafe: StakingInterface {
```

Comment:
Move force_unstake also here.

Reply:
How about `StakingUnchecked`? "unsafe" makes me feel unsafe :)

Reply:
I like `StakingUnchecked` way more.

Reply:
What do you think about StakingInsecure since it either removes security (deposit) or does not care about one.